.changeset/nine-chicken-cheat.md

```diff
@@ -0,0 +1,5 @@
+---
+"@medusajs/medusa": patch
+---
+
+fix(medusa): align prices formating in the import & export strategies
```

Comment:
todo: I only see the export here, should we also update the import as mentioned in the changeset? So that you can set the numeric amount in the import and it will be stored in cents?

cc @olivermrbl

Reply:
The PR description reads: _"to align format with import strategy which expects prices to be in a human-readable format"_, so I assumed this was the case.

@fPolic can you pitch in? :)

Reply:
With the import strategy, we are expecting the price to be in a human-readable format already

Reply:
Perfect 👍